athena-snowflake/src/main/java/com/amazonaws/athena/connectors/snowflake/SnowflakeMetadataHandler.java

```diff
@@ -173,7 +174,7 @@ public void getPartitions(BlockWriter blockWriter, GetTableLayoutRequest getTabl
         else {
             double totalRecordCount = 0;
             LOGGER.info(COUNT_RECORDS_QUERY);
-            List<String> parameters = Arrays.asList(getTableLayoutRequest.getTableName().getTableName());
+            List<String> parameters = Arrays.asList(getTableLayoutRequest.getTableName().getSchemaName(), getTableLayoutRequest.getTableName().getTableName());
```

Comment:
Related to line #184: Do you know why we only store the `totalRecordCount` of the latest ResultSet?

Shouldn't it be sum of the RowCount?

Reply:
Hi, The query is a metadata query, to fetch the total records count for a table in snowflake. It will return single row with row_count value being retrieved.